src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -1495,4 +1609,51 @@ public Window.WindowOrientation WindowOrientation
             }
         }
     }
+
+    /// <summary>
+    /// Move Completed event is sent when window has been moved the display server.
+    /// It is triggered by calling RequestMoveToServer().
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WindowMoveCompletedEventArgs : EventArgs
+    {
+        private Position2D position;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Position2D WindowCompletedPosition
+        {
+            get
+            {
+                return position;
+            }
+            set
```

Comment:
I don't think the setter of event args properties need to be public.
How about add a constructor such as `WindowCompletedEventArgs(Position2D position)` and make setter private?
```C#
public Position2D WindowCompletedPosition
{
    get => position;
    private set => position = value;
}
```

Reply:
You're right, We will update that with your comment. 
Thank you for detail review. :)